packages/wdio-utils/src/driver/utils.ts

```diff
@@ -84,7 +84,8 @@ export const downloadProgressCallback = (artifact: string, downloadedBytes: numb
         return
     }
     const percentage = ((downloadedBytes / totalBytes) * 100).toFixed(2)
-    log.info(`Downloading ${artifact} ${percentage}%`)
+    // @ts-expect-error Invoking custom loglevel
```

Comment:
mhm .. I wonder if we can make the logger export with the `progress` method so we don't have to do this. Given we might want to use this one elsewhere in the future it might make sense to extend the types.

Reply:
Agreed, I'll start figuring a way on extending the types to include 'progress'.